tests/Simplify/PlusPlusTest.elm

```diff
@@ -134,6 +134,176 @@ a = "" ++ "a"
                             }
                             |> Review.Test.whenFixed """module A exposing (..)
 a = "a"
+"""
+                        ]
+        , test """should replace "a" ++ "b" by "ab" if they're on the same line""" <|
+            \() ->
+                """module A exposing (..)
+a = "a" ++ "b"
+"""
+                    |> Review.Test.run ruleWithDefaults
+                    |> Review.Test.expectErrors
+                        [ Review.Test.error
+                            { message = "++ on string literals can be turned into a single string literal"
+                            , details = [ "Try moving all the elements into a single string literal." ]
+                            , under = "++"
+                            }
+                            |> Review.Test.whenFixed """module A exposing (..)
+a = "ab"
+"""
+                        ]
+        , test """should replace "a" ++ "b" ++ x by "ab" ++ x""" <|
+            \() ->
+                """module A exposing (..)
+a = "a" ++ "b"
```

Comment:
missing the ++ x

Reply:
And I've now introduced a new bug. Adding a test case and fixing it.